### PR TITLE
feat: article loading state

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

**Comment by trevorah** [comment]
😍

File: android/app/build.gradle [hunk 1]
```diff
@@ -144,6 +144,7 @@ repositories {
 }
 
 dependencies {
+    compile project(':react-native-linear-gradient')
```
Comment (CoreFloDev) [inline android/app/build.gradle]: implementation please

Comment (fampinheiro) [reply android/app/build.gradle]: What do you mean by implementation?

Comment (CoreFloDev) [reply android/app/build.gradle]: compile is obsolete in gradle 4.0, you need to change it to implementation instead. Check for more info https://developer.android.com/studio/build/gradle-plugin-3-0-0-migration.html#new_configurations

**Comment by craigbilner** [comment]
@fampinheiro looks good.

Can we:

- [ ] fix the paging `n - zero`
- [ ] add avatar to stories
- [ ] remove the redundant empty state story

File: packages/author-profile/author-profile-author-head.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import AuthorHead from "@times-components/author-head";
+import Gradient from "@times-components/gradient";
+import React from "react";
+import { StyleSheet, View } from "react-native";
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: "center",
+    backgroundColor: "#F9F8F3",
+    flexDirection: "column",
```
Comment (craigbilner) [inline packages/author-profile/author-profile-author-head.js]: default?

File: packages/author-profile/author-profile-content.js [hunk 3]
```diff
@@ -1,116 +1,115 @@
...
-            name={name}
-            bio={biography}
-            uri={uri}
-            title={jobTitle}
-            twitter={twitter}
-            onTwitterLinkPress={onTwitterLinkPress}
+    const data = articlesLoading
+      ? Array(pageSize)
+          .fill()
+          .map((_, id) => ({
```
Comment (craigbilner) [inline packages/author-profile/author-profile-content.js]: minor: but the Haskellesque placeholder if used as a pattern can be confused for `lodash`?

File: packages/author-profile/author-profile-author-head.js (hunk 2 above)
Comment (fampinheiro) [reply packages/author-profile/author-profile-author-head.js]: It was here when we had conditional styles. Not the case anymore (removing).

File: packages/author-profile/author-profile-content.js (hunk 3 above)
Comment (fampinheiro) [reply packages/author-profile/author-profile-content.js]: Can change to a meaningful name. 👍

**Comment by fampinheiro** [comment]
@craigbilner regarding the pagination issue.
We didn't reach a conclusion if the pagination should be attached to the author-head query or a separated one, or try to hack something with state on the `author-profile` component.

**Comment by craigbilner** [comment]
@fampinheiro thought we were going to look at the Apollo update function which will retain the given page size?

**Comment by fampinheiro** [comment]
I read about the [update](http://dev.apollodata.com/react/cache-updates.html#directAccess) but this is used with a mutation.

**Pull request merged**